### ISSUE TITLE
feature: allow push to current branch

### ISSUE THREAD

**Issue by JetteBke** [Parfuemerie-Douglas/scaffolder-backend-module-azure-repositories#11]
Hello everybody,

We are using trunk based development and want to allow pushing the changes made by the template to the main branch.
I want to push to the current branch from the `azure:repo:push` action.
Currently, this fails with `the branch already exists` because it tries to create a new one.
I have created a fix for it but am not allowed to create branches in this project.
Please find the code snippet below in case you prefer to add it yourself. Otherwise, I am happy to submit a PR if you give me permission.

`helpers.ts`

```
... 
 const currentBranch = await git.currentBranch({dir})

  if (currentBranch !== branch) {
    await git.branch({
      dir,
      ref: branch,
    })
  
    await git.checkout({
      dir,
      ref: branch,
    });
  }
...
```

Let me know if you need more info :)
Thanks!

**Comment by hevans-dglcom** [comment]
@JetteBke  you can fork the project, create the changes in your tree and then PR back against our tree.

Have a try and let me know if you need help :)

**Comment by JetteBke** [comment]
hi @hevans-dglcom , I am gonna try that, thank you! :)

**Comment by JetteBke** [comment]
hi again @hevans-dglcom , I added a PR: #13 
I didn't find a good way to test this locally, but it should be fine.
Let me know if it works.

**Comment by hevans-dglcom** [comment]
@JetteBke we have released 0.2.5 with your changes included. Let us know if it works as expected.

**Comment by JetteBke** [comment]
Thank you so much @hevans-dglcom , it worked! 🥳
